testpilot/frontend/static-src/app/views/experiment-tour-dialog-view.js

```diff
@@ -30,7 +30,7 @@ export default BaseView.extend({
         <div class="tour-actions">
           <div class="tour-back"></div>
           <div class="tour-next"></div>
-          <button data-hook="cancel-modal-done" class="button default large no-display" data-l10n-id="tourDoneButton">Done</button>
+          <button data-hook="complete-modal-done" class="button default large no-display" data-l10n-id="tourDoneButton">Done</button>
```

Comment:
[Need to rename this data-hook in the test for this view](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/test/views/experiment-tour-dialog-view.js#L39), also.